build.gradle

```diff
@@ -11,7 +11,8 @@ buildscript {
         classpath 'com.android.tools.build:gradle:3.6.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
```

Comment:
I think this should be removed

Reply:
Removed https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version